charts/zabbix/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 ---
-apiVersion: v2 # Don't change this
+apiVersion: v2  # Don't change this
 name: zabbix
-version: 6.1.2 # helm chart version
+version: 6.1.2  # helm chart version
```

Comment:
Hi @fibbs!

I suggest changing the helm chart version to 7.0.0. This change can be made in a new PR before publishing a new release.

Reply:
Didn't we agree that we are going to use the first digit after the dot for major releases? This specific diff is not because of version change (which would happen outside PR) but because of the space before comment. Did linting with `ct` tool, which demands more than one space before in-line comment

Reply:
Okay @fibbs. I remember the deal. I thought I'd put more emphasis on this break change... but we can keep the deal.